webui/react/src/shared/components/Spinner/Spinner.test.tsx

```diff
@@ -32,56 +57,41 @@ describe('Spinner', () => {
     document.body.appendChild(style);
   });
 
-  it('blocks inner content while spinning', async () => {
```

Comment:
**question**: Why is this test being removed?

Reply:
Good question. `Spin` originally supports this feature https://4x.ant.design/components/spin/, so this test case is redundant, and it's flaky

Reply:
now its not flaky somehow, so get it back.

Reply:
nvm its still flaky. it works on local, but not on CI

Reply:
https://github.com/testing-library/user-event/issues/662#issuecomment-904365493 fixed the issue